### PR TITLE
feat(connector): [FIUU] PR template code 

### PR DESCRIPTION
**Pull request by awasthi21**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Template code added for new connector FIUU

### Additional Changes

- [x] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [x] This PR modifies application configuration/environment variables

## How did you test it?
Template PR

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: crates/hyperswitch_connectors/src/default_implementations.rs [hunk 1]
```diff
@@ -86,6 +86,7 @@ macro_rules! default_imp_for_authorize_session_token {
 }
 
 default_imp_for_authorize_session_token!(
+    connectors::Fiuu,
```
Comment (SamraatBansal) [inline crates/hyperswitch_connectors/src/default_implementations.rs:89]: Reorder alphabetically all the enums

File: crates/hyperswitch_connectors/src/default_implementations_v2.rs [hunk 2]
```diff
@@ -762,7 +783,53 @@ default_imp_for_new_connector_integration_revoking_mandates!(
...
+//             Authentication,
+//             types::ExternalAuthenticationFlowData,
+//             types::authentication::ConnectorAuthenticationRequestData,
+//             types::authentication::AuthenticationResponseData,
+//         > for $path::$connector
+//         {}
+//         impl
+//             services::ConnectorIntegrationV2<
+//             PreAuthentication,
+//             types::ExternalAuthenticationFlowData,
```
Comment (SamraatBansal) [inline crates/hyperswitch_connectors/src/default_implementations_v2.rs:810]: Is this required?

File: api-reference-v2/openapi_spec.json [hunk 3]
```diff
@@ -10283,17 +10283,6 @@
...
-            "required": [
-              "paypal"
-            ],
-            "properties": {
-              "paypal": {
-                "type": "object"
-              }
-            }
-          },
           {
```
Comment (SamraatBansal) [inline api-reference-v2/openapi_spec.json:10286]: Get this checked with @SanchithHegde

**Review by SamraatBansal — CHANGES_REQUESTED** [review]

File: crates/hyperswitch_connectors/src/default_implementations.rs (hunk 1 above)
Comment (awasthi21) [reply crates/hyperswitch_connectors/src/default_implementations.rs]: done

File: crates/hyperswitch_connectors/src/default_implementations_v2.rs (hunk 2 above)
Comment (awasthi21) [reply crates/hyperswitch_connectors/src/default_implementations_v2.rs]: removed

File: api-reference-v2/openapi_spec.json (hunk 3 above)
Comment (awasthi21) [reply api-reference-v2/openapi_spec.json]: Took the latest pull, it wasn;t there

**Review by deepanshu-iiitu — APPROVED** [review]

**Review by SamraatBansal — APPROVED** [review]

**Review by SanchithHegde — APPROVED** [review]

**Review by sahkal — APPROVED** [review]

**Review by deepanshu-iiitu — APPROVED** [review]

**Review by sahkal — APPROVED** [review]

**Review by sai-harsha-vardhan — APPROVED** [review]

**Review by SanchithHegde — APPROVED** [review]

**Pull request merged**